### ISSUE TITLE
NS7 Angular

### ISSUE THREAD

**Issue by keerl** [nativescript-community/ui-image#31]
I'm not sure if this plugin officially supports NS7 + Angular 10 yet, but I am getting an Angular import error when trying to use `@nativescript-community/ui-pager@latest`.

```
ERROR in ../node_modules/@nativescript-community/ui-image/angular/fesm2015/nativescript-community-ui-image-angular.js
Module not found: Error: Can't resolve '@nativescript/angular/element-registry' in '/Users/karlschmaltz/Desktop/Projects/test-app/node_modules/@nativescript-
community/ui-image/angular/fesm2015'
 @ ../node_modules/@nativescript-community/ui-image/angular/fesm2015/nativescript-community-ui-image-angular.js 2:0-73 30:0-15
 @ ./app/app.module.ts
 @ ./main.ts
```

I believe this fix may be as easy as replacing all `@nativescript/angular/XXXXX` imports with just `@nativescript/angular` but I'm not 100% sure.

**Comment by farfromrefug** [comment]
@keerl should be fixed in 4.0.7

**Comment by keerl** [comment]
Works! Thanks
